CHANGELOG.md

```diff
@@ -67,9 +67,14 @@ _Code:_
 
   - Fixed multiple buffer overflows in Secure Message ([#763](https://github.com/cossacklabs/themis/pull/763)).
 
+- **C++**
+
+  - `themispp::secure_message_t` now produces signed messages of optimal length ([#775](https://github.com/cossacklabs/themis/pull/775)).
+
 - **Go**
 
   - Error `ErrOverflow` is now deprecated in favor of `ErrOutOfMemory`, new error types were added ([#711](https://github.com/cossacklabs/themis/pull/711)).
+  - `SecureMessage` now produces signed messages of optimal length ([#775](https://github.com/cossacklabs/themis/pull/775)).
```

Comment:
```suggestion
  - `SecureMessage` now produces signed messages of optimal length, encrypted data by SecureMessage might be smaller than before ([#775](https://github.com/cossacklabs/themis/pull/775)).
```

Reply:
I've updated this (and C++) passage to read:

> `SecureMessage.Sign()` output is a bit smaller now

This communicate the effect of the change to the users, and if they are really interested *why* the output is smaller, they can look into this PR.